### PR TITLE
Update 02_Functions.md

### PR DESCRIPTION
**Pull request by Deep1307**
Added code to print the result of function multiply and accordingly added the documentation line for it.

### CODE REVIEW

File: examples/01_introduction/02_Functions.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ fun main() {
     printMessageWithPrefix("Hello")                                     // 7
     printMessageWithPrefix(prefix = "Log", message = "Hello")           // 8
     println(sum(1, 2))                                                  // 9
+    println(multiply(2, 4))                                             // 10
```
Comment (Deep1307) [inline examples/01_introduction/02_Functions.md:26]: Added this line of code to Print the result of the function multiply.

File: examples/01_introduction/02_Functions.md [hunk 2]
```diff
@@ -34,7 +35,8 @@ fun main() {
 6. Calls the function with two parameters, passing values for both of them.
 7. Calls the same function omitting the second one. The default value `Info` is used. 
 8. Calls the same function using [named arguments](https://kotlinlang.org/docs/reference/functions.html#named-arguments) and changing the order of the arguments.
-9. Prints the result of a function call.
+9. Prints the result of a function call (of function `sum`).
+10. Prints the result of a function call (of function `multiply`).
```
Comment (p7nov) [inline examples/01_introduction/02_Functions.md:39]: Let's shorten the text a little: 

```suggestion
9. Prints the result of the `sum` function call.
10. Prints the result of the `multiply` function call.
```

**Review by p7nov — CHANGES_REQUESTED** [review]
Hi! 
Thank you for this improvement.
Please see my suggestion for simplifying the text.

**Comment by p7nov** [comment]
Thanks!

**Comment by Deep1307** [comment]
> Thanks!

I am really glad to contribute. This is my first contribution so far. I am just getting started to contribute to the community.

**Comment by Deep1307** [comment]
> Thanks!

@p7nov  Can you tell me, How can I contribute more to the community and organization?

**Comment by p7nov** [comment]
> > Thanks!
> 
> @p7nov Can you tell me, How can I contribute more to the community and organization?

It's always nice to hear such things from people!
There is a page [Contributing to Kotlin](https://kotlinlang.org/contribute.html) on kotlinlang.org. Here you can find ways to help the development of Kotlin.

**Comment by Deep1307** [comment]
> > > Thanks!
> > 
> > 
> > @p7nov Can you tell me, How can I contribute more to the community and organization?
> 
> It's always nice to hear such things from people!
> There is a page [Contributing to Kotlin](https://kotlinlang.org/contribute.html) on kotlinlang.org. Here you can find ways to help the development of Kotlin.

@p7nov  Thank You

**Pull request merged**